src/version.js

```diff
@@ -9,14 +9,23 @@ function trackNewVersion({
   newVersion,
 }) {
   let range = new semver.Range(newRange);
+
   if (range.set.length > 1) {
     console.warn(`Current range has an OR (${name} ${oldRange}) and is too hard to increment, falling back to ^`);
     newRange = `^${newVersion}`;
   } else if (range.set[0].length === 1) {
-    newRange = newVersion;
+    let raw = range.raw.trim();
+
+    // wildcards remain the same
+    if (raw === '*' || raw === '') {
```

Comment:
`range.range === ''` should be the only thing you need to check

Reply:
how? I could find no documentation on this :-\

turns out it's just _super_ hidden: https://github.com/npm/node-semver/blob/master/test/ranges/to-comparators.js#L10-L12